src/foam/dao/EasyDAO.js

```diff
@@ -388,6 +426,16 @@ return delegate;
       generateJava: false,
       name: 'decorators'
     },
+    {
+      class: 'FObjectArray',
+      of: 'foam.core.PropertyInfo',
+      name: 'order'
+    },
+    {
+      class: 'FObjectArray',
+      of: 'foam.core.PropertyInfo',
+      name: 'index'
```

Comment:
Just adding on index isn't very useful. Better if this took an array of indices (each of which is an array of PropertyInfos.

Reply:
@kgrgreer What would the serviceScript setIndex call look like?